UniversityofHelsinki/ResearchDataServiceChannel
# Wizard POC

To test:

composer update-deps
drush cim

- Verify that /wizard is accessible (without content for now)
- Verify that vocabulary "hy_wizard" is present (if cim fails, module creates it for you)
- Add terms to the taxonomy (f.ex "foo", "bar" and "xoo")
- Verify that /wizard now displays these terms
- Verify that clicking a term goes to subpage (without any content for now) and back button is showing
- Verify that back button goes back to the main listing
- Add multiple levels of taxonomy and in the last one, add a service to it
- Verify that traversing to said taxonomy term, the service is displayed in a box.

There are know issues like REST api returns taxonomy terms in different order when pushing "back". Also service link is not yet connected.
